app/src/main/java/org/ieeeguc/ieeeguc/models/User.java

```diff
@@ -95,6 +98,64 @@ public String getCommitteeName() {
         return committeeName;
     }
 
+    public static void ForgetPassword(String email, final HTTPResponse HTTPR) {

```

Comment:
add documentation to your method look at @rehamayman96 code.
Follow the naming conventions of variables